src/sentry/static/sentry/app/views/ruleEditor/ruleNode.jsx

```diff
@@ -52,8 +52,10 @@ class RuleNode extends React.Component {
           <span ref="html" dangerouslySetInnerHTML={{__html: node.html}} />
         </RuleNodeForm>
         <RuleNodeControls>
-          <a onClick={this.props.onDelete}>
-            <span className="icon-trash" />
+          <a onClick={this.props.onDelete} className="button button-default button-sm">
```

Comment:
Why no `Button`?

Reply:
Button gets <enter> behavior which i wanted to avoid

Reply:
does adding `type="button"` solve it?

Reply:
Sort of. But I still prefer the behavior of <a> which is harder to trigger. For example you can't tab to a link.

Reply:
`tabIndex=-1`?

Reply:
that works!